vignettes/datatable-intro.Rmd

```diff
@@ -52,8 +52,8 @@ options(width = 100L)
 ```
 
 ```{r}
-input <- if (file.exists("flights14.csv")) {
-   "flights14.csv"
+input <- if (file.exists("flights14.csv.bz2")) {
```

Comment:
I would rather prefer to keep intro vignette simple and very basic, without need for suggested deps. Extra note is not really a problem as long as we handle it well in CI.